packages/ra-ui-materialui/src/list/filter/FilterForm.tsx

```diff
@@ -30,35 +24,45 @@ import { FilterFormInput } from './FilterFormInput';
 import { FilterContext } from '../FilterContext';
 
 export const FilterForm = (props: FilterFormProps) => {
-    const { defaultValues, filters: filtersProps, ...rest } = props;
+    const { filters: filtersProps, ...rest } = props;
 
     const { setFilters, displayedFilters, filterValues } = useListContext();
     const filters = useContext(FilterContext) || filtersProps;
 
-    const mergedInitialValuesWithDefaultValues =
-        mergeInitialValuesWithDefaultValues(
-            defaultValues || filterValues,
-            filters
-        );
```

Comment:
This whole call is actually useless, because:
1. `defaultValues` don't work
2. `mergeInitialValuesWithDefaultValues` actually does nothing, since there is a runtime check to prevent having both `alwaysOn` and `defaultValue`. It's probably a historic tech debt, that's why I removed it.